### PR TITLE
fix: Updating alerting email subject for easy email filtering.

### PR DESCRIPTION
**Pull request by deepak1725**
# Description
There is a difficulty in Gmail filtering due to spaces in Subject.
In this PR we are replacing " "(spaces) with | (Pipes) for easy filtering.

**CSSRE Jira**: https://issues.redhat.com/browse/CSSRE-1572  
**MGDAPI Jira**: https://issues.redhat.com/browse/MGDAPI-2368

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

## Checklist
- [x] Verified independently on a cluster by reviewer

### CODE REVIEW

**Comment by deepak1725** [comment]
image

Find Test eml here: https://drive.google.com/file/d/1p5wWmn9QW76mCL-aZN7qUMKyXD1nBL3n/view?usp=sharing

**Comment by deepak1725** [comment]
Multiple Alerts
image

Test EML: 
- https://drive.google.com/file/d/1aeWUMPhrB2YF1Tojy91l56tclv9_IsAZ/view?usp=sharing 
- https://drive.google.com/file/d/1BCQR6bU28WQ8DfATMg0YgnI9djrvvJ7b/view?usp=sharing

File: templates/monitoring/alertmanager/alertmanager-email-config.tmpl [hunk 1]
```diff
@@ -5,11 +5,11 @@
 {{ define "__intlysubject" }}
 	
 	{{/* declaring variables */}}
-	{{ $lenOfAlerts := ( len .Alerts ) }} {{ $clustername := "${CLUSTER_NAME}"}} {{ $alertname := "" }} {{ $severity := "" }} {{ $ifCritical := "" }}
+  {{ $lenOfAlerts := ( len .Alerts ) }} {{ $clustername := "${CLUSTER_NAME}"}} {{ $alertname := "" }} {{ $severity := "" }} {{ $ifCritical := "" }}
```
Comment (austincunningham) [inline templates/monitoring/alertmanager/alertmanager-email-config.tmpl:8]: why this tab change ?

Comment (deepak1725) [reply templates/monitoring/alertmanager/alertmanager-email-config.tmpl]: fixed

**Comment by austincunningham** [comment]
/lgtm
I can see that the spaces get replace with | in the subject in the example email included in the pr.

**Pull request merged**